Dockerfile.jsonschema

```diff
@@ -1,6 +1,6 @@
 # 3.18.2
 FROM alpine@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1
-RUN apk add --update protoc=3.21.12-r2 protobuf-dev=3.21.12-r2 go=1.20.5-r0 git=2.40.1-r0
+RUN apk add --update protoc=3.21.12-r2 protobuf-dev=3.21.12-r2 go=1.20.7-r0 git=2.40.1-r0
```

Comment:
CC @znewman01 for consensus, but I'd personally be okay with removing these pins now that we understand how they interact with the surrounding alpine image: the image itself has a "world" pin that effectively pins each of these dependencies.

(In other words, the pins here are extraneous and only cause trouble when the alpine image is updated automatically and they aren't.)

Reply:
Ahhh okay. That makes me a little nervous about auto-updates of the image itself. But I guess any big changes due to updates would get caught by the generation tests.

So I say go ahead and remove